src/queries/query-string-query.js

```diff
@@ -1,13 +1,20 @@
 /**
  * Construct a Query String query.
  *
+ * @param  {Array}  fields  The field names to query over.
  * @param  {String} term  Query value.
  * @return {Object}       Query String query.
  */
-export default function queryStringQuery(term) {
-  return {
+export default function queryStringQuery(fields, term) {
+  let query = Array.isArray(fields) ? term : fields;
+  fields = Array.isArray(fields) ? fields : [];
+  let queryString = {
     query_string: {
-      query: term
+      query: query
```

Comment:
Because of [es2015 enhanced object literals](https://github.com/lukehoban/es6features#enhanced-object-literals) I think you can just do:
```
let queryString = {
  query
}
```
:tada: